src/lib/CombatCalc.ts

```diff
@@ -247,11 +264,11 @@ export default class CombatCalc {
 
   private isWearingSilverWeapon(): boolean {
     return this.wearing([
```

Comment:
This is my bad for not doing it originally, but can you think of a good way to add silver bolt detection here? Similar to below for leaf-bladed I'm fine with not checking that we're using an applicable weapon for now.

Reply:
I've added a check for silver bolts + ranged style before this broader gear check block.